### ISSUE TITLE
error: invalid path

### ISSUE THREAD

**Issue by ericjdz** [Virtual-Protocol/agent-contributions-library#120]
![Screenshot 2024-09-02 183539](https://github.com/user-attachments/assets/ee8c16a7-5fc7-40b7-a13e-d8233eba9f6a)

I'm still having problems with cloning the repo

**Comment by Serv08** [comment]
What happens when you run `git restore --source=HEAD :/` ?

**Comment by armlynobinguar** [comment]
Yes yes, do git restore @ericjdz

**Comment by Serv08** [comment]
@armlynobinguar , I think there's an error when cloning in Windows. I also faced this similar problem on Windows but not in Ubuntu. I'm not sure why but this happens when I used `git restore --source=HEAD :/`

It delete some parts of the repo. Maybe the cause of the problem last night where some of the files were gone and someone pushed it. 
![image](https://github.com/user-attachments/assets/e66dc279-2e93-4d27-8aed-70294f028e4b)

What I did was use `git restore --staged *` to revert this deletion. This will discard the deletion done when cloning it.

![image](https://github.com/user-attachments/assets/5319e411-ca08-4474-9b2a-291b7b2f52b8)

However, another error is present. Some files can't be staged back.
```bash
deleted contribution_datasets/amber/readme.md
deleted contribution_datasets/ansem/readme.md
deleted contribution_datasets/colonel ray /readme.md
deleted contribution_datasets/dingaling /readme.md
deleted contribution_datasets/mia khalifa/readme.md
deleted contribution_datasets/yellow panther /readme.md
```

Upon checking from [stack overflow](https://stackoverflow.com/questions/63727594/github-git-checkout-returns-error-invalid-path-on-windows), it suggested to use `git config core.protectNTFS false`.

Then after that, try `git restore --staged *` again then check the folder where the clone was. Manually check if all the files are there.

It seems that the extra space after `colonel ray `, `dingaling `, and `yellow panther ` is a problem for Windows, but not for other OS.
